spec/requests/lettings_logs_controller_spec.rb

```diff
@@ -1153,119 +1144,54 @@
+              log
             end
 
-            it "redirects to review page" do
-              expect(response).to redirect_to("/lettings-logs/#{completed_lettings_log.id}/review")
-            end
-          end
-
-          context "with lettings logs from a closed previous collection period" do
             before do
-              sign_in user
-              Timecop.freeze(2023, 2, 1)
-              Singleton.__init__(FormHandler)
-              get "/lettings-logs/#{completed_lettings_log.id}", headers:, params: {}
-            end
-
-            after do
-              Timecop.return
-              Singleton.__init__(FormHandler)
+              get lettings_log_path(lettings_log)
             end
 
             it "redirects to review page" do
-              expect(response).to redirect_to("/lettings-logs/#{completed_lettings_log.id}/review")
+              expect(response).to redirect_to("/lettings-logs/#{lettings_log.id}/review")
             end
 
             it "displays a closed collection window message for previous collection year logs" do
-              get "/lettings-logs/#{completed_lettings_log.id}", headers:, params: {}
-              expect(completed_lettings_log.form.new_logs_end_date).to eq(Time.zone.local(2022, 11, 20))
-              expect(completed_lettings_log.status).to eq("completed")
               follow_redirect!
-              expect(page).to have_content("This log is from the 2021/2022 collection window, which is now closed.")
+              expect(page).to have_content(/This log is from the \d{4}\/\d{4} collection window, which is now closed\./)
             end
           end
 
-          context "when a lettings log is for a renewal of supported housing, property information does not need to show" do
-            let(:lettings_log) do
-              FactoryBot.create(
-                :lettings_log,
-                owning_organisation: user.organisation,
-                managing_organisation: user.organisation,
-                assigned_to: user,
-                startdate: Time.zone.now,
-                renewal: 1,
-                needstype: 2,
-                rent_type: 3,
-                postcode_known: 0,
-              )
-            end
-
-            before do
-              sign_in user
-            end
+          context "when a lettings log is for a renewal of supported housing" do
+            let(:lettings_log) { create(:lettings_log, :startdate_today, assigned_to: user, renewal: 1, needstype: 2, rent_type: 3, postcode_known: 0) }
 
-            around do |example|
-              FormHandler.instance.use_real_forms!
-              example.run
-              FormHandler.instance.use_fake_forms!
+            it "does not show property information" do
+              get lettings_log_path(lettings_log)
+              expect(page).to have_content "Tenancy information"
+              expect(page).not_to have_content "Property information"
             end
```

Comment:
I added this test in since it was specified in the name of the context block it's in but not tested.